src/coreComponents/mesh/ElementType.hpp

```diff
@@ -109,6 +109,9 @@ ENUM_STRINGS( ElementType,
               "HendecagonalPrism",
               "Polyhedron" );
 
+/// String available for mesh errors
+inline auto constexpr generalMeshErrorAdvice = "\nPlease consider checking the validity of your mesh with the `mesh_doctor` GEOS python tools.";
```

Comment:
If possible, could you add the URL `https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/pythonTools/mesh_doctor.html`?
Also, could you remove the initial `\n`?